src/Bicep.Core/TypeSystem/Az/ResourceTypeProviderFactory.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Bicep.Types;
+using Azure.Bicep.Types.Az;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Features;
+using Bicep.Core.Modules;
+using Bicep.Core.Registry.Oci;
+using Bicep.Core.Semantics.Namespaces;
+using Bicep.Core.TypeSystem.Az;
+using Newtonsoft.Json;
+
+namespace Bicep.Core.TypeSystem
+{
+
```

Comment:
nit: multiple blank lines

Reply:
should be resolver by `dotnet format`, marking as resolved